docs/pages/components/button.md

```diff
@@ -132,10 +83,63 @@ Group a series of buttons together on a single line with the segmented button.
+
+{% capture button-menu %}
+<button class="fd-button fd-button--menu">Action Button</button>
+<button class="fd-button fd-button--standard fd-button--menu">Standard Button</button>
+<button class="fd-button fd-button--ghost fd-button--menu">Ghost Button</button>
+<button class="fd-button fd-button--positive fd-button--menu">Positive Button</button>
+<button class="fd-button fd-button--negative fd-button--menu">Negative Button</button>
+<button class="fd-button fd-button--attention fd-button--menu">Attention Button</button>
+<br><br>
+<button class="fd-button fd-button--menu" aria-disabled="true" disabled>Action Button</button>
+<button class="fd-button fd-button--standard fd-button--menu" aria-disabled="true" disabled>Standard Button</button>
+<button class="fd-button fd-button--ghost fd-button--menu" aria-disabled="true" disabled>Ghost Button</button>
+<button class="fd-button fd-button--positive fd-button--menu" aria-disabled="true" disabled>Positive Button</button>
+<button class="fd-button fd-button--negative fd-button--menu" aria-disabled="true" disabled>Negative Button</button>
+<button class="fd-button fd-button--attention fd-button--menu" aria-disabled="true" disabled>Attention Button</button>
+<br><br>
+<button class="fd-button fd-button--emphasized fd-button--menu sap-icon--cart">Add to Cart</button>
+<button class="fd-button fd-button--menu sap-icon--cart">Add to Cart</button>
+<button class="fd-button fd-button--transparent fd-button--menu sap-icon--cart">Add to Cart</button>
+<button class="fd-button fd-button--emphasized fd-button--menu fd-button--positive sap-icon--accept">Approve</button>
+<button class="fd-button fd-button--negative fd-button--menu sap-icon--decline">Reject</button>
+<button class="fd-button fd-button--attention fd-button--menu sap-icon--warning">Attention</button>
+<br><br>
+<button class="fd-button fd-button--menu sap-icon--cart"></button>
+<button class="fd-button fd-button--transparent fd-button--menu sap-icon--cart"></button>
+<button class="fd-button fd-button--standard fd-button--menu sap-icon--filter"></button>
+<button class="fd-button fd-button--menu fd-button--positive sap-icon--accept"></button>
+<button class="fd-button fd-button--menu fd-button--negative sap-icon--decline"></button>
+<button class="fd-button fd-button--menu fd-button--attention sap-icon--warning"></button>
+<br><br>
+<button class="fd-button fd-button--menu">Default</button>
+<button class="fd-button fd-button--compact fd-button--menu">Compact</button>
+{% endcapture %}
+{% include display-component.html component=button-menu %}
+
+### Split Menu Button
+
+It is separate into 2 areas: the text label and the icon. The separator between them signals that the two areas result in different actions. The user has two choices: activating the text label on the button triggers the action. Activating the arrow opens the menu. The split button consolidates a variety of commands, especially when one of the commands is used more often.
```

Comment:
The menu can also be split into 2 areas: the text label and the icon. The separator between them signals that the two areas result in different actions. The user has two choices: 1- activating the text label on the button triggers the action or 2- activating the arrow opens the menu. The split button consolidates a variety of commands, especially when one of the commands is used more often.